docs/guides/deployment/deploy-to-fly-io-with-sqlite.md

```diff
@@ -1,13 +1,117 @@
-# Deploy to Fly.io with SQLite
+# Deploy Platformatic Applications to Fly.io
 
-:::note
+## Deploying a Platformatic Runtime Application 
+
+This guide provides instructions on deploying a Platformatic Runtime application to Fly.io. With a runtime application, you are deploying your entire application, including all services in the `services` folder.
+
+
+### Dockerfile for Runtime Application
+
+Here is an example Dockerfile for a Platformatic Runtime application:
+
+```dockerfile
+FROM node:20-alpine
+
+ENV APP_HOME=/home/app/node/
+
+WORKDIR $APP_HOME
+
+COPY . .
+COPY package-lock.json package-lock.json ./
+RUN npm ci
+
+RUN npx platformatic compile
+
```

Comment:
In this way they ship all _development_ dependencies in their container. They should prune for production after compilation.

Reply:
I see, thanks.

Reply:
That step is missing

Reply:
I've added a multi-build stage for this, also added it to use production for the docker image.

Reply:
This does not do a multi-stage build, take a look at https://github.com/platformatic/platformatic/blob/main/Dockerfile on how to do one.